docs/tutorial.rst

```diff
@@ -1000,6 +1000,31 @@ separately from Zarr.
 
 .. _tutorial_copy:
 
+Accessing Zip Files on S3
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The built-in `ZipStore` will only work with paths on the local file-system, however
```

Comment:
I'm not sure how links work in sphinx, but could we make this a link to the API docs for `ZipStore`?